src/modeler/AutoRest.Swagger/Validation/AvoidMsdnReferences.cs

```diff
@@ -21,6 +21,18 @@ public class AvoidMsdnReferences : TypedRule<string>
         /// </summary>
         public override ValidationCategory ValidationCategory => ValidationCategory.SDKViolation;
 
+        /// <summary>
+        /// What kind of open api document type this rule should be applied to
+        /// </summary>
+        public override ServiceDefinitionDocumentType ServiceDefinitionDocumentType => ServiceDefinitionDocumentType.ARM;
```

Comment:
A little unsure actually, feels like something ARM recommended but also think that the documentation team were involved in requesting this rule.